src/main/java/org/trigon/hardware/SignalThreadBase.java

```diff
@@ -30,6 +46,11 @@ public void updateLatestTimestamps() {
         Logger.processInputs(name, threadInputs);
     }
 
+    /**
+     * Gets the latest timestamps.
```

Comment:
latest timestamps of what?

Reply:
the timestamps aren't used/updated anywhere. So unless I'm missing smth, rn they're latest timestamps of nothing.

Reply:
They are definitely updated
Dig more